docs/src/main/paradox/docs/releases/v1.3-to-v1.4-migration.md

```diff
@@ -0,0 +1,190 @@
+    projection_id text PRIMARY KEY,
+    progress text
+) WITH bloom_filter_fp_chance = 0.01
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'NONE'}
+    AND comment = ''
+    AND compaction = {'class': 'org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy', 'max_threshold': '32', 'min_threshold': '4'}
+    AND compression = {'chunk_length_in_kb': '64', 'class': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND crc_check_chance = 1.0
+    AND dclocal_read_repair_chance = 0.1
+    AND default_time_to_live = 0
+    AND gc_grace_seconds = 864000
+    AND max_index_interval = 2048
+    AND memtable_flush_period_in_ms = 0
+    AND min_index_interval = 128
+    AND read_repair_chance = 0.0
+    AND speculative_retry = '99PERCENTILE';
+
+CREATE TABLE IF NOT EXISTS delta.projections_failures (
+    projection_id text,
+    offset text,
+    persistence_id text,
+    sequence_nr bigint,
+    value text,
+    PRIMARY KEY (projection_id, offset, persistence_id, sequence_nr)
+) WITH CLUSTERING ORDER BY (offset ASC, persistence_id ASC, sequence_nr ASC)
+    AND bloom_filter_fp_chance = 0.01
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'NONE'}
+    AND comment = ''
+    AND compaction = {'class': 'org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy', 'max_threshold': '32', 'min_threshold': '4'}
+    AND compression = {'chunk_length_in_kb': '64', 'class': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND crc_check_chance = 1.0
+    AND dclocal_read_repair_chance = 0.1
+    AND default_time_to_live = 0
+    AND gc_grace_seconds = 864000
+    AND max_index_interval = 2048
+    AND memtable_flush_period_in_ms = 0
+    AND min_index_interval = 128
+    AND read_repair_chance = 0.0
+    AND speculative_retry = '99PERCENTILE';
+
+CREATE TABLE IF NOT EXISTS delta_snapshot.snapshots (
+  persistence_id text,
+  sequence_nr bigint,
+  timestamp bigint,
+  ser_id int,
+  ser_manifest text,
+  snapshot_data blob,
+  snapshot blob,
+  meta_ser_id int,
+  meta_ser_manifest text,
+  meta blob,
+  PRIMARY KEY (persistence_id, sequence_nr))
+  WITH CLUSTERING ORDER BY (sequence_nr DESC) AND gc_grace_seconds =864000
+  AND compaction = {
+    'class' : 'SizeTieredCompactionStrategy',
+    'enabled' : true,
+    'tombstone_compaction_interval' : 86400,
+    'tombstone_threshold' : 0.2,
+    'unchecked_tombstone_compaction' : false,
+    'bucket_high' : 1.5,
+    'bucket_low' : 0.5,
+    'max_threshold' : 32,
+    'min_threshold' : 4,
+    'min_sstable_size' : 50
+    };
+```      
+
+6. Deploy the new service images (tag `1.4`) for [delta](https://hub.docker.com/r/bluebrain/nexus-delta) with the `MIGRATE_V13_TO_V14=true` and `REPAIR_FROM_MESSAGES=true` environment variables.
+This will instruct the service to migrate all the messages table from iam, admin and kg into the delta keyspace. Afterwards the `tag_views` table and their related tables will be properly initialized.
```

Comment:
- It is not clear if the old tables are deleted afterwards.
- Maybe we should warn that we need twice the disk space used by Cassandra during the migration ?
- Redeploy the service afterwards without the flags ?